compose.yaml

```
@@ -0,0 +1,46 @@
+version: '3.8'
+
+services:
```

```suggestion
services:
```

Docker compose considers version numbers a deprecated property.
https://github.com/docker/compose/issues/11628#issuecomment-2000072567